stage1/init/pod.go

```diff
@@ -168,10 +160,10 @@ func (p *Pod) appToSystemd(ra *schema.RuntimeApp, interactive bool) error {
 	}
 
 	env := app.Environment
-	env.Set("AC_APP_NAME", name)
+	env.Set("AC_APP_NAME", appName.String())
```

Comment:
@yifan-gu Yes, I think you're probably right. Or just AC_IMAGE_NAME since AC_ACI is a bit redundant.